packages/Tokens/package.json

```diff
@@ -16,10 +16,8 @@
     "pretranspile": "node processTokens.js"
   },
   "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
     "node-sass": "^4.9.2",
```

Comment:
Do you mind moving these two to devDeps?

Reply:
okiie